nemo/collections/nlp/modules/common/transformer/text_generable_model.py

```diff
@@ -0,0 +1,60 @@
+from builtins import NotImplementedError, bool, int, float, str
+from typing import List, TypedDict, Union
+
+from torch import Tensor
+
+
+class SamplingParam(TypedDict):
+    max_length: int  # The maximum length of the sequence to be generated.
```

Comment:
should we add an option to stop if "." was generated?

Reply:
i think compute_logprobs should be replaced by generate(compute_logprobs=True) or something like that